rest/route/distro.go

```diff
@@ -788,7 +788,7 @@ func (h *distroIcecreamConfigHandler) Parse(ctx context.Context, r *http.Request
 // Run enqueues a job to run a script on all hosts that are not down for the
 // given given distro ID.
 func (h *distroIcecreamConfigHandler) Run(ctx context.Context) gimlet.Responder {
-	hosts, err := host.Find(ctx, (host.ByDistroIDsOrAliasesRunning(h.distro)))
+	hosts, err := host.Find(ctx, host.ByDistroIDsOrAliasesRunning(h.distro))
```

Comment:
Ty for removing the random parentheses (no change)! 🙏

Reply:
i think this was just my editor being clever but i'll take the credit 😂